test/compose2image.py

```diff
@@ -40,7 +40,9 @@ def composeurl2images(
             if len(subvariantmatch) > 0:
                 candidates = subvariantmatch
 
-    return [(composepath + qcow2[0].path) for qcow2 in candidates]
+    image_urls = [(composepath + qcow2[0].path) for qcow2 in candidates]
+    return sorted(image_urls, reverse=True)
```

Comment:
How does this guarantee that the latest image is the first one in the list?

Reply:
@richm There is no 100% guarantee. This implementation expects the new nightly compose to have the same format as the older one and to have alphabetically bigger character in the image name. So it uses only `sorted()` function.